### DISCUSSION TITLE
Should `schemaVersion` be named something else?

### DISCUSSION THREAD

**Discussion by nickkaczmarek** [wwt/WorkflowSchema#5] [Q&A] [answered]
We're able to use a property (`$schema`) in the `workflow-schema.json` file to denote the schema it should be validated against. And this is awesome, but it's a little confusing now that we have `$schema` and `schemaVersion`. I don't know a great name off the top, but perhaps we should think of something.

**Comment by Tyler-Keith-Thompson** [comment, marked as answer]
You know we debated this a bit. The only alternative posed was `specificationVersion` which is both harder to type and perhaps slightly more ambiguous.

**Reply by nickkaczmarek** [reply]
After more thought I think keeping it is fine. We can always revisit this if it comes up again.
